ee/presencedetection/presencedetection.go

```diff
@@ -0,0 +1,51 @@
+package presencedetection
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+const DetectionFailedDurationValue = -1 * time.Second
+
+type PresenceDetector struct {
+	lastDetectionUTC          time.Time
```

Comment:
if this is `time.Time` maybe drop the `UTC`?

Reply:
will drop it, but we still want UTC instead of local time I'm assuming

Reply:
Generally it's better to keep things in UTC, yes. And when we emit to k2, we should use one of the standard formats, or just integer unix time

Reply:
cool, we don't emit the time for this to K2, just the go style duration since last detection like `0s` or `1m30s`